src/components/ArticleCard.jsx

```diff
@@ -1,24 +1,26 @@
 import React from "react";
 import { Placeholder, Grid } from "semantic-ui-react";
-import { Link } from 'react-router-dom';
+import { Link } from "react-router-dom";
 
 const ArticleCard = ({ article }) => {
   return (
     <Grid.Row width={3} centered>
       <Placeholder
-        style={{ height: 250, width: 400 }}
+        fluid
+        style={{ height: 300, width: 600 }}
         key={article.id}
         id={"article-" + article.id}
       >
         <Placeholder.Image />
 
         <Link
           to={{
-            pathname: `/article/${article.id}`
+            pathname: `/article/${article.id}`,
           }}
-          
-        > 
-          <h5 id="article-title" style={{ textAlign: "center" }}>{article.title}</h5>
+        >
+          <h5 id="article-title" style={{ textAlign: "center" }}>
+            {article.title}
+          </h5>
         </Link>
       </Placeholder>
     </Grid.Row>
```

Comment:
Theses changes are out of scope, am I correct? The PR contains 18 files changed, so try to limit unnecessary changes.

Reply:
The face lift of the styling.

Reply:
it was part of the chore styling

Reply:
I see, that's okay. Though we should probably avoid including chores in large PRs in the future.